utils/make-packages.sh

```diff
@@ -8,6 +8,9 @@
 # same location.
 scriptdir=$(dirname $(realpath $0))
 
+# Get the location of the rpm builder script.
+rpmDir=$(dirname $(realpath $scriptdir))/rpm
```

Comment:
$scriptdir is already canonical, so this can be simplified to just `rpmDir=$(dirname $scriptdir)/rpm`